backend/wmg/pipeline/summary_cubes/expression_summary/load.py

```diff
@@ -57,8 +59,16 @@ def build_in_mem_cube(
         vals["sum"][idx : idx + n_vals] = cube_sum[cube_idx, mask]
         vals["nnz"][idx : idx + n_vals] = cube_nnz[cube_idx, mask]
 
+        dataset_index = 0
+
         for i, k in enumerate(other_cube_attrs):
-            vals[k][idx : idx + n_vals] = attr_values[i]
+            if k == "dataset_id":
+                dataset_index = i
+            if k != "publication_citation":
+                vals[k][idx : idx + n_vals] = attr_values[i]
+
+        dataset_dict = return_dataset_dict_w_publications()
+        vals["publication_citation"][idx : idx + n_vals] = dataset_dict(attr_values[dataset_index])
```

Comment:
`dataset_dict[attr_values[dataset_index]]`

Use `datastic_dict[]` instead of `dataset_dict()`